pkg/slo-controller/nodeslo/nodeslo_cm_event_handler.go

```diff
@@ -49,6 +49,7 @@ type SLOCfg struct {
 	ResourceQOSCfgMerged extension.ResourceQOSCfg       `json:"resourceQOSCfgMerged,omitempty"`
 	CPUBurstCfgMerged    extension.CPUBurstCfg          `json:"cpuBurstCfgMerged,omitempty"`
 	SystemCfgMerged      extension.SystemCfg            `json:"systemCfgMerged,omitempty"`
+	ExtensionCfgMerged   extension.ExtensionCfg         `json:"extensionCfgMerged,omitempty"` // for third-party extension
```

Comment:
what if there are multi extensions, it seems that the format of ExtensionCfgMerged should be map

Reply:
Strategy in Cfg is a map. 
`// +k8s:deepcopy-gen=true
type ExtensionCfg struct {
	ClusterStrategy *slov1alpha1.ExtensionsMap `json:"clusterStrategy,omitempty"`
	NodeStrategies  []NodeExtensionStrategy    `json:"nodeStrategies,omitempty"`
}

// +k8s:deepcopy-gen=true
type NodeExtensionStrategy struct {
	NodeCfgProfile             `json:",inline"`
	*slov1alpha1.ExtensionsMap // for third-party extension
}`

Reply:
It can reuse the  *slov1alpha1.ExtensionsMap struct.